CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+Thanks for considering contributing and making our planet easier to explore!
+
+We're excited you would like to contribute to Cumulus! Whether you're finding bugs, adding new features, fixing anything broken, or improving documentation, get started by submitting an issue or pull request!
+
+## Submitting an Issue
+
+If you have any questions or ideas, or notice any problems or bugs, first [search open issues](https://github.com/cumulus-nasa/cumulus-message-adapter-js/issues) to see if the issue has already been submitted. We may already be working on the issue. If you think your issue is new, you're welcome to [create a new issue](https://github.com/cumulus-nasa/cumulus-message-adapter-js/issues/new).
+
+## Pull Requests
+
+If you want to submit your own contributions, follow these steps;
+
+* Fork the Cumulus repo
```

Comment:
Change this to "this repo" or the name of this repo.

Reply:
Fixed.